src/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
```

Comment:
This is the actual schema, I realize running multiple scrapes will duplicate the data but that's a problem for later.